src-tauri/src/capturer/mod.rs

```diff
@@ -130,6 +130,14 @@ pub async fn stop_capture(app_handle: AppHandle) {
 
     println!("Preview path: {:?}", preview_path);
 
+    // Hide cropper, create editor
```

Comment:
This needs to happen much sooner.. ideally the first thing to happen as soon as this funciton is fired. Right now the UI still feels gimmicky to use and I often end up clicking the stop-recording button multiple times.